### ISSUE TITLE
NER hyperparameters

### ISSUE THREAD

**Issue by NPoe** [dmis-lab/biobert#90]
Hi, 

I have been trying to reproduce your results on the 8 NER datasets, so far with limited success. A couple of questions:

1) Would it be possible to publish the optimal hyperparameters for each NER dataset? That would save me (and probably others) quite a bit of superfluous work.
2) Since you train on train and dev combined (train_dev.tsv), which dataset do you use for hyperparameter tuning?
3) In your paper, you mention lr, batch size and number of epochs. Are there any other hyperparameters to consider? What was the hyperparameter range for number of epochs?
4) I noticed that my results vary from run to run by quite a bit. Did you average test set F1 over runs or did you pick the best run (and if so, based on what dataset)?

**Comment by jhyuklee** [comment]
Hi, please see [here](https://github.com/dmis-lab/biobert/issues/94) for hyperparamter tuning(Q1, Q3). We did not average the F1 scores but reported the best one (Q4) based on the hyperparameters chosen from validation sets (Q2). Thanks.
